base/gcutils.jl

```diff
@@ -259,4 +259,13 @@ function enable_logging(on::Bool=true)
     ccall(:jl_enable_gc_logging, Cvoid, (Cint,), on)
 end
 
+"""
+    GC.is_logging()
```

Comment:
The name makes it sound like it's querying whether logging is currently happening but it's actually querying whether logging is enabled. Perhaps `GC.logging_enabled()` would be a more accurate name? It matches the C global being accessed, plus it mirrors other functions that query whether something is enabled, e.g. `inlining_enabled` and `coverage_enabled` in `Core.Compiler`.

Reply:
fine by me. changed